cosmos/cosmos.github.io
# Issue #7: add components from tendermint site

Fix #7 

## Description
* Add bios to cosmos.network/team

## Steps to test
* Check out branch
* Go to http://localhost:8700/team
* Make sure it all looks reasonable

# Todos
- [x] Update @jaekwon's bio
- [ ] Get feedback from @jaekwon regarding how he wants to display the link to All In Bits
